new-log-viewer/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "webpack --config webpack.prod.js",
     "lint": "npm run lint:check",
-    "lint:ci": "npm run lint:check -- --output-file eslint-report.json --format json",
+    "lint:ci": "npm run lint:check -- --max-warnings 0",
```

Comment:
Why not make `--max-warnings 0` the default for `lint:check`?

Reply:
Sounds better